src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -129,6 +130,10 @@ public function load(array $configs, ContainerBuilder $container): void
             ->addTag('api_platform.filter');
 
         if ($container->hasParameter('test.client.parameters') && class_exists(AbstractBrowser::class)) {
+            if (!class_exists(HttpClient::class)){
+                throw new RuntimeException('Run "composer require symfony/http-client" to use the test client"');
```

Comment:
```suggestion
                throw new RuntimeException('Run "composer require --dev symfony/http-client" to use the test client"');
```

Can you also rebase and run PHP CS Fixer please?

Reply:
Done :)